_data/hosting_providers.json

```
@@ -129,6 +129,16 @@
     "reviewed": "2019.5.31",
     "note": ""
   },
+  {
+    "name": "netclusive",
+    "link": "https://netclusive.de/",
+    "category": "full",
+    "tutorial": "https://www.netclusive.de/faq/content/36/430/de/kann-ich-meine-webseite-ssl-verschluesseln.html",
+    "announcement": "",
+    "plan": "",
+    "reviewed": "2020.10.01",
+    "note": "The website is in german. Old plans of shared web hosting with confixx panel have full HTTPS support. New plans of shared web hosting with Plesk panel also have full HTTPS support."
```

```suggestion
    "note": "The website is in German."
```

Since all the plans seem to now have full support, there's no need to additionally indicate this, so let's remove it for clarity.